epictrack-web/src/components/AppHelpButton/SupportCenterMenuItem.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import helpPageMap from "./HelpPageMap.json";
+import { HelpMenuItem } from "./HelpMenuItem";
+
+type HelpPageLink = {
+  epicTrackPath: string;
+  helpPage: string;
+};
+
+const SupportCenterMenuItem = () => {
+  const handleClick = () => {
+    const currentPath = window.location.pathname.split("?")[0]; // Get pathname without query params
+    const currentPathSegments = currentPath.split("/"); // Split current path
```

Comment:
we might not get this path parameter for selecting the help page every time. For instance, if we are on work plan for Project Notification, we should be pointing to the help page which talks about Project Notification where the URL would be https://epictrack-dev.apps.gold.devops.gov.bc.ca/work-plan?work_id=58
This way would fit for regular pages.

Reply:
how would you figure from this https://epictrack-dev.apps.gold.devops.gov.bc.ca/work-plan?work_id=58 that it should be directed to Project Notification help?